code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -60,6 +60,17 @@
 		/obj/item/device/analyzer,
 		/obj/item/taperoll/engineering,
 		/obj/item/weapon/extinguisher/mini)
+		
+/obj/item/weapon/storage/belt/utility/ert

```

Comment:
This needs to be:

> /obj/item/weapon/storage/belt/utility/full/New()

Since you want it to spawn the items it in when it's created. Doing this will also clear out all the duplication errors you are getting in the travis build.